src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -162,9 +166,10 @@ impl SharedBufferBatchInner {
     }
 
     fn get_table_key_ends(
+        table_id: TableId,
         mut range_tombstone_list: Vec<DeleteRangeTombstone>,
-    ) -> (bool, Vec<u8>, Vec<u8>, Vec<DeleteRangeTombstone>) {
-        let mut largest_table_key = vec![];
+    ) -> (bool, Vec<u8>, Bound<Vec<u8>>, Vec<DeleteRangeTombstone>) {
```

Comment:
This return result is too complex. I prefer using a struct to replace tuple

Reply:
> This return result is too complex. I prefer using a struct to replace tuple

fixed